NEW_MODULE_TECH_EVAL.MD

```diff
@@ -0,0 +1,73 @@
+
+# New Module Technical Evaluation Processes
+
+## Overview
+This document outlines the processes for **technical** evaluation of module for inclusion in a FOLIO release.  The following activities are covered:
+1. Submission
+1. Evaluation
+1. Review
+1. Feedback
+1. Approval/Rejection
+
+### Supporting documentation
+* Details of the JIRA workflow are captured in a separate document:  (insert link). 
+* Technical evaluation is part of a larger process involving the other FOLIO Councils (PC/CC).  A description of the full process can be found here:  (insert link). 
+
+## Submission
+1. The Product Council fills out a form in JIRA (using a template) and submits it to the Technical Council.  
+    * See JIRA workflow for details.
+    * The Technical Council will only respond to submissions from the Product Council.
+1. The Technical Council will update the JIRA within 1 week (See JIRA workflow for details) with the following information:
+    * Confirm receipt of the submission
+    * Who will be performing the evaluation
+    * Provide an rough estimate for the initial evaluation to be completed
+        * Should be 3 weeks or less.
+    * Request additional information
+        * If there are missing points of contact, the Product Council will be contacted
+        * Any questions about the module will be directed to the points of contact.
+
+### Submission form
+Information gathered includes:
+* Module name
+* Description of module
+* Link to source - a tag/commit in a publicly accessible git (preferably GitHub) repository 
+* List of related modules (e.g. is this a Frontend/backend pair?  Business logic/storage pair?  etc.)
+* Points of contact (Subject Matter / Domain / Technical)
+  * Each point of contact's role should be clearly identified.
+  * A single, main point of contact must have an account in issues.folio.org JIRA.
+* Free form section for notes, clarifications, background, links to supporting documentation, etc.
+* Link to self evaluation results
+* Informational Section - provides info, instead of eliciting it.
+  * A link/pointer to the most recent version of the Acceptance Criteria.
+  * A maximum duration (3 weeks) from the submission date for the initial review.  
+    * This does not include activities that happen after feedback is given, etc. 
+  * The applicable deadline for submissions in the release timeline (not the date, but the name of the milestone/deadline e.g. "deadline for new modules")
```

Comment:
Some bullet points end in a full stop, others don't.

Is there a reason for this distinction?